<launch>

  <node name="serial_node" pkg="rosserial_python" type="serial_node.py">
    <param name="port" value="/dev/ttyACM0" />
    <param name="baud" value="115200" />
  </node>
  <node name="synchronizer" pkg="detection" type="synchronizer"/>

  <include file="$(find openni2_launch)/launch/openni2.launch"/>

  <node name="filter" pkg="detection" type="filter" output="screen">
    <param name="camera_topic" value="/camera/ir/image_raw" />
    <param name="arduino_topic" value="/brightness" />
  </node>  

  <include file="$(find apriltag_ros)/launch/continuous_detection.launch">
    <arg name="camera_name" value="/camera/ir/"/>
    <arg name="image_topic" value="/image_filtered"/>
  </include>

</launch>  
